Dockerfile

```diff
@@ -0,0 +1,32 @@
+FROM ubuntu:16.04
+MAINTAINER Sharath Ganesh Pai <sharath@cliqz.com>
+ENV DEBIAN_FRONTEND noninteractive
+RUN dpkg --add-architecture i386 && \
+    apt-get update && \
+    apt-get upgrade -y && \
+    apt-get install -y autoconf2.13 build-essential ccache curl git lib32z1 \
+        libc6:i386 libncurses5:i386 libstdc++6:i386 mercurial openjdk-8-jdk \
+        python-dev python-pip python-setuptools unzip uuid \
+        wget xz-utils zip && \
+    apt-get clean -y && \
+    rm -rf /var/lib/apt/lists/* /tmp/* /var/tmp/* && \
+    mkdir -p /root/.mozbuild/proguard/lib && \
+    wget --output-document=/tmp/proguard.tgz --quiet "https://vorboss.dl.sourceforge.net/project/proguard/proguard/5.3/proguard5.3.3.tar.gz" && \
```

Comment:
This is external dependencies, which can die at any moment. Better to copy to our S3 bucket.

Reply:
True. I will copy it to S3.

Reply:
also proguard is in ubuntu repository, at least it is in 17.10